api_backend_server/tests/unit_tests/pytest_activities_apis.py

```diff
@@ -0,0 +1,70 @@
+from api_backend_server.web_app.apis.activities import _convert_unix_to_iso8610, _parse_to_utc_iso8610
+from api_backend_server.web_app.apis.activities import create_activity, get_user_activities
+import pytest
+import datetime
+import mock as mock
+from mock import MagicMock
+
+
+def test__parse_to_utc_iso8610_should_succeed():

```

Comment:
Test fails - you are comparing a datetime object with a string, so assert will fail :stuck_out_tongue: